compiler/codegen/cg-expr.cpp

```diff
@@ -6299,6 +6299,44 @@ DEFINE_PRIM(BREAKPOINT) {
   }
 }
 
+DEFINE_PRIM(CONST_ARG_HASH) {
+  INT_ASSERT(call->numActuals() == 1);
+  INT_ASSERT(call->get(1)->isRefOrWideRef());
+  GenRet arg = call->get(1);
+
+  if (call->get(1)->isWideRef()) {
+    Symbol* sym = call->get(1)->typeInfo()->getField("addr", true);
```

Comment:
I found it confusing that this line is here but not used in the sequence just below. Could you either move this line to just before the `codegenSizeof` where it is used; or do the `codegenSizeof` here and store it in a variable for use on that later line?